pkg/operator/operator.go

```diff
@@ -246,6 +247,9 @@ func (o *Operator) Start(ctx context.Context) {
 		wg.Add(1)
 		go func() {
 			defer wg.Done()
+			// Taking the first supported NodeClass to be the default NodeClass
+			ctx = injection.NodeClassToContext(ctx, cp.GetSupportedNodeClasses())
```

Comment:
We shouldn't inject all of the NodeClasses, right? We should just inject the first one

Reply:
I'm planning on using the nodeClass information to fill the gap of APIVersion for when converting between v1 -> v1beta1